src/components/Dashboard/PositionsTable/PositionsTable.js

```diff
@@ -196,6 +198,17 @@ const PositionsTable = (props) => {
     excludeDataTableColumn,
   } = usePositionDataTableCompose(positionsFiltered, confirmAction);
 
+  /**
+   *
+   * @param {UserPositionsCollection} allPositions positions collection.
+   * @returns {Number} count of positions in last 24 hour.
+   */
+  const filterPastDayPositions = (allPositions) => {
```

Comment:
Since this method returns a number of positions, maybe it can be called something like `pastDayPositionsCount` ?
For functions that are not so obvious to understand we can add JSDoc description above @param, but by changing the name it'd be enough.

Reply:
@cwagner22  Done.